answerking_app/tests/test_orderlines.py

```diff
@@ -172,13 +174,10 @@ def test_update_existing_orderline_invalid_returns_bad_request(self):
 
     def test_update_existing_orderline_negative_returns_bad_request(self):
         # Arrange
-        expected_failure_error: ErrorMessage = {
-            "error": {
-                "message": "Resource update failure",
-                "details": "Failed to add item to order",
-            }
+        expected_failure_error: dict[str, list[str]] = {
```

Comment:
type needs to be changed to SerializerQuantityError

Reply:
thats what I was thinking too. if only :'(

Reply:
but I have already created SerializerQuantityError so should probably do it as it is done elsewhere

Reply:
I think we should just remove it too then 😅, for the items there are just too many combinations

Reply:
that is true, I'm happy with that solution